getting-started.md

```diff
@@ -59,3 +59,23 @@ $ exogress spawn --access-key-id=<ACCESS_KEY_ID> --secret-access-key=<SECRET_ACC
 ```
 
 That's it. Now the server on port `3000` is available under the selected domain.
+
+## Advanced
+
+### Project-level config
+
+Project-level config simplifies configuration in cases when you have multiple client-level configs and want to set some common behaviour across all of them. Does not support upstreams.
+
+### Certificates
+
+We use LetsEncrypt to issue and renew TLS certificates for your domains. Make sure the status is `Resolved` before sending the traffic to the domain.
```

Comment:
status should be `Active` or `ready`, depending on where you look at it. Resolved related to DNS